client/app/scripts/superdesk-desks/aggregate.js

```diff
@@ -45,7 +44,7 @@
                 }));
         }))
         .then(angular.bind(this, function() {
-            return api.query('saved_searches', {}, session.identity)
+            return api.query('all_saved_searches', {})
```

Comment:
@ioanpocol FYI: Vince created this task for us to revert this to user specific saved searches (SD-3138) so I created a PR for it.